pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -106,6 +106,17 @@
         "CRX",
         "CRY",
         "CRZ",
+        "C(PauliX)",
+        "C(PauliY)",
+        "C(PauliZ)",
+        "C(Hadamard)",
+        "C(S)",
+        "C(T)",
+        "C(PhaseShift)",
+        "C(RX)",
+        "C(RY)",
+        "C(RZ)",
+        "C(Rot)",
```

Comment:
Where do we test these controlled wires with 1 and multiple controlled wires and values?

Reply:
these are tested [here](https://github.com/PennyLaneAI/pennylane-lightning/blob/4dd344ed5d1f02fecbd56be0c0910a78c601f701/tests/test_gates.py#L452) in python, c++ tests will be added